reference/configuration/swiftmailer.rst

```diff
@@ -191,6 +237,14 @@ logging
 If true, Symfony's data collector will be activated for Swift Mailer and
 the information will be available in the profiler.
 
+.. tip::
+
+    The following configuration options are compatible with ``%env()%`` parameters:
+    ``url``, ``transport``, ``username``, ``password``, ``host``,
+    ``port``, ``timeout``, ``source_ip``, ``local_domain``,
+    ``encryption``, ``auth_mode``.
```

Comment:
Considering https://github.com/symfony/swiftmailer-bundle/pull/159#issuecomment-283193936, we should remove these two parameters for now, wdyt?